src/store/getters/state-getters.ts

```diff
@@ -225,7 +227,8 @@ export const isIncorporationAgreementValid = (state: any): boolean => {
 export const isApplicationValid = (state: any): boolean => {
   return (state.stateModel.defineCompanyStep.valid && state.stateModel.addPeopleAndRoleStep.valid &&
     state.stateModel.createShareStructureStep.valid && state.stateModel.incorporationDateTime.valid &&
-    state.stateModel.incorporationAgreementStep.valid && state.stateModel.certifyState.valid)
+    state.stateModel.incorporationAgreementStep.valid &&
+    (state.stateModel.certifyState.valid && state.stateModel.certifyState.certifiedBy))
```

Comment:
This looks almost like a duplicate of `isEnableFilePay`...
- should this evaluation check if incorp date time is valid?
- could `isEnableFilePay = return isApplicationValid` ? Or maybe just use a single getter?

Reply:
Will look at this!

Reply:
Yea, this is a good spot for refactoring. I'm going to remove `isEnabledFilePay` to replace with isApplicationValid. The latter makes more sense now that the FilePay btn is always technically enabled.

Reply:
Updated in next commit.
They were near duplicates.

Reply:
Sounds good!